pcsx2/DEV9/ATA/Commands/ATA_SCE.cpp

```diff
@@ -49,7 +59,33 @@ void ATA::SCE_IDENTIFY_DRIVE()
 {
 	PreCmd();
 
-	//HDD_IdentifyDevice(); //Maybe?
+	// fill sceSec response with default data
+	memcpy(sceSec, "Sony Computer Entertainment Inc.", 32);
+	memcpy(sceSec + 0x20, "SCPH-20401", 10);
+	memcpy(sceSec + 0x30, "  40", 4);
+
+	sceSec[0x40] = 0; // 4-byte HDD internal SCE serial, always starts with 0x02
+	sceSec[0x41] = 0;
+	sceSec[0x42] = 0;
+	sceSec[0x43] = 0x02;
+
+	sceSec[0x46] = 0x1a; // unknown 8-bytes
+	sceSec[0x47] = 0x01;
+	sceSec[0x48] = 0x02;
+	sceSec[0x49] = 0x20;
+	sceSec[0x4c] = 0x01;
```

Comment:
Add a line between `sceSec[0x49]` and `sceSec[0x4c]` so it's clear at a glance that 0x4A,0x4B are skipped.

Reply:
Ah yes, thanks, forgot about that, will fix it.